dev_apps/in_app_pay_test/index.html

```diff
@@ -0,0 +1,47 @@
+<html>
+  <head>
+    <title>In-App Payments Tester</title>
+    <meta charset="utf-8"/>
+    <link rel="stylesheet" type="text/css" href="css/index.css"/>
+  </head>
+  <body>
+    <div id="app">
+      <div id="jwt-input">
+        <select name="server" id="server">
+          <option value="0">
+            marketplace.firefox.com (prod)
```

Comment:
on hindsight, allowing an editable JWT to test payments on production seems like a terrible idea. However, I looked at the corresponding key on the server and it does not seem to be active. Choosing this option will cause an error.